ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -262,7 +262,12 @@ func createCluster(context *cli.Context, awsClients *AWSClients, commandConfig *
 	}
 
 	// Populate cfn params
-	cfnParams, err := cliFlagsToCfnStackParams(context, commandConfig.Cluster, launchType, containerInstanceTaggingSupported)
+	var cfnParams *cloudformation.CfnStackParams
+	if containerInstanceTaggingSupported {
+		cfnParams, err = cliFlagsToCfnStackParams(context, commandConfig.Cluster, launchType, tags)
+	} else {
+		cfnParams, err = cliFlagsToCfnStackParams(context, commandConfig.Cluster, launchType, nil)
+	}
```

Comment:
For readability you could refactor this so that just `tags` is set within the conditional, and then you can set 
`cfnParams, err := cliFlagsToCfnStackParams(context, commandConfig.Cluster, launchType, tags)` outside the condition.